lib/zlib/adler32.js

```diff
@@ -3,6 +3,25 @@
 // Note: adler32 takes 12% for level 0 and 2% for level 6.
 // It doesn't worth to make additional optimizationa as in original.
 // Small size is preferable.
+//
+// (C) 1995-2013 Jean-loup Gailly and Mark Adler
+// (C) 2014-2017 Vitaly Puzrin and Andrey Tupitsin
+//
+// This software is provided 'as-is', without any express or implied
```

Comment:
Original source are made by original author that are not binded by license.... 

You could strip at build time if needed